script/configs/temp_exclude_excerpt.yaml

```diff
@@ -0,0 +1,19 @@
+# Packages that have not yet adopted code-excerpt.
+#
+# This only exists to allow incrementally adopting the new requirement.
+# Packages shoud never be added to this list.
+
+# TODO(ecosystem): Remove everything from this list. See
```

Comment:
fwiw generally for TODOs the name in brackets is the name of the person with context (not the name of the person responsible for fixing it)

Reply:
Fixed